src/components/Board.js

```diff
@@ -3,29 +3,32 @@ import './Board.css';
 import Square from './Square';
 import PropTypes from 'prop-types';
 
-
+// This turns the 2D array into a 1D array
 const generateSquareComponents = (squares, onClickCallback) => {
-  // Complete this for Wave 1
-  // squares is a 2D Array, but 
-  //  you need to return a 1D array
-  //  of square components
-
-}
+  const singleArraySquares = [].concat(...squares);
+  return singleArraySquares.map((square) => {
+    return (
+      <Square
+        value={square.value}
+        id={square.id}
+        onClickCallback={onClickCallback}
+        key={square.id}
+      />
+    );
+  });
+};
 
 const Board = ({ squares, onClickCallback }) => {
   const squareList = generateSquareComponents(squares, onClickCallback);
-  console.log(squareList);
-  return <div className="grid" >
-    {squareList}
-  </div>
-}
+  return <div className='grid'>{squareList}</div>;
+};
 
 Board.propTypes = {
   squares: PropTypes.arrayOf(
     PropTypes.arrayOf(
       PropTypes.shape({
         id: PropTypes.number.isRequired,
-        value: PropTypes.string.isRequired
+        value: PropTypes.string.isRequired,
       })
     )
```

Comment:
you can add this PropType for your onClickCallback `onClickCallback: PropTypes.func.isRequired,`